config/styleguide.config.js

```diff
@@ -37,23 +36,34 @@ module.exports = {
     // key is a path to match, value is the name to show in the styleguide for the import statement
     const namespacedComponents = {
       FlexGrid: 'FlexGrid',
-      StepTracker: 'Steps',
       'ExpandCollapse/Panel': 'ExpandCollapse',
-      'Typography/Text': 'Text',
       DisplayHeading: 'DisplayHeading',
       Heading: 'Heading',
+      'Text/TextSup': 'Text',
+      'TextArea/TextArea': 'TextArea',
+      'StepTracker/Steps': 'StepTracker',
     }
 
     const componentDirectory = path.dirname(componentPath)
 
     const componentPathTest = Object.keys(namespacedComponents).find(pathTest =>
       componentDirectory.includes(pathTest)
     )
+
     if (componentPathTest) {
       name = namespacedComponents[componentPathTest]
     }
 
-    return `import { ${name} } from '@telusdigital/tds'`
+    let kebabizeName = name
+      .split(/(?=[A-Z])/)
+      .join('-')
+      .toLowerCase()
+
+    if (name === 'Accordion') {
+      kebabizeName = 'expand-collapse'
+    }
+
+    return `import { ${name} } from '@tds/core-${kebabizeName}'`
```

Comment:
Most of our packages are not named exports, so using deconstruction notation for packages such as `{ FlexGrid }` is invalid. Though I kept a static list of named exports, is there a better way to detect these? https://github.com/telusdigital/tds-codemod/blob/bee9356cd0031457b12c5e92be2d797692a6801a/components.js

I suggest keeping a list of exceptions (namely, ExpandCollapse) to print as named exports.

![image](https://user-images.githubusercontent.com/12798751/36753394-0664cb0e-1bd4-11e8-9e90-c0fad4671941.png)

Reply:
Yep, I believe everything is the default export except for ExpandCollapse and Accordion.